src/main/java/controller/MainController.java

```diff
@@ -39,6 +40,17 @@
 	private ExcelImporter ei;
 	private ArrayList<String[]> excelRows;
 	private ArrayList<ExcelRow> excelRowsConverted = new ArrayList<ExcelRow>();
+	private ArrayList<CodeQualityRule> rulesList = new ArrayList<CodeQualityRule>();
+
+	/**
+	 * MainController constructor. Creates the default rules to be used.
+	 */
+	public MainController() {
+		CodeQualityRule is_long_method = new CodeQualityRule("is_long_method", "LOC > 80 &6 CYCLO > 10", true, false);
```

Comment:
A little mistake in the creation of the CodeQualityRule objects, you used &6 instead of &&